README.md

```diff
@@ -5,23 +5,54 @@ This repo provides a data exporter by which Kubernetes operators can leverage th
 ## Installation
 
 We provide a helm chart to help install the exporter. Note that this data exporter ONLY retrieves the carbon intensity data from 
-[WattTime](https://www.watttime.org/). You need to get the **authentication ID/Password** from WattTime organization before using the exporter.
+[WattTime](https://www.watttime.org/) OR [Electricity Maps](https://www.electricitymaps.com/). 
+
+### WattTime
+
+You need to get the **authentication ID/Password** from WattTime organization before using the exporter.
+
+```bash
+export WT_USERNAME=XXXX   # WattTime auth info.
+export WT_PASSWORD=YYYY
+export REGION=westus     # The region where the AKS cluster locates.
+
+helm del carbon-intensity-exporter
+helm install carbon-intensity-exporter \
+   --set carbonDataExporter.region=$REGION \
+   --set wattTime.username=$WT_USERNAME \
+   --set wattTime.password=$WT_PASSWORD \
+   ./charts/carbon-intensity-exporter
+```
+
+### Electricity Maps
+
+You need to get an **API token** from Electricity Maps before using the exporter.
+You can check the name of the API token HTTP header to use and the base URL in
+the Electricity Maps API portal.
 
 ```bash
-export WTUSERNAME=XXXX   # WattTime auth info.
-export WTPASSWORD=YYYY
+export EM_API_TOKEN=XXXX   # Electricity Maps API token.
+export EM_API_TOKEN_HEADER=auth-token   # Electricity Maps API token HTTP header.
+export EM_BASE_URL=https://api.electricitymap.org/v3/
```

Comment:
The base URL needs to be configurable if the user is using the Electricity Maps API portal.

e.g.  https://api-access.electricitymaps.com/2w97h07rvxvuaa1g

Reply:
Perhaps we should include a note that the trailing slash needs to be in the URL. I tested this with a free trial of ElectricityMaps and my `api-acces` base URL did not work until I added the trailing slash to it.